### DISCUSSION TITLE
Unlock and lock the door from CLI

### DISCUSSION THREAD

**Discussion by toshibochan** [bimmerconnected/bimmer_connected#306] [Feature request] [answered]
how to unlock and lock the door?

**Comment by scstraus** [comment]
If you enabled the option to allow locking and unlocking and gave the VIN of the car to enable it in the API setup, the integration should create a lock entity with which you can call the service lock.lock or lock.unlock with to lock and unlock the car.

**Reply by toshibochan** [reply]
You can give me example

**Reply by toshibochan** [reply]
How to enable that option?

**Reply by scstraus** [reply]
Sorry, I assumed you were using homeassistant. It seems that may not be the case, in which case I can't help much. But if you want to try it with homeassistant, I'd be happy to help, though the instructions on the standard integration are quite clear IMO.

**Comment by rikroe** [comment, marked as answer]
Currently door locking/unlocking cannot be done using the `bimmerconnected` cli interface.

The options are:
- Use the Home Assistant integration
- Write your own little Python script that imports `bimmerconnected` and execute the service from there
- Create a PR that enhances `cli.py` with the `LOCK`/`UNLOCK` like done `VEHICLE_FINDER` here: https://github.com/bimmerconnected/bimmer_connected/commit/5ca622c9a6975a59a61c56d87a6521c9e2564b01#diff-a80f5bcba1e13046252f218bc4ccdd3b2bdbf9e6d1fe2089504729629d37cf58

**Reply by toshibochan** [reply]
any plan to integrate lock/unlock in the future?
